plugins/mtu/plugin.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018-present the CoreDHCP Authors. All rights reserved
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package mtu
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/insomniacslk/dhcp/dhcpv4"
+
+	"github.com/coredhcp/coredhcp/handler"
+	"github.com/coredhcp/coredhcp/logger"
+	"github.com/coredhcp/coredhcp/plugins"
+)
+
+var log = logger.GetLogger("plugins/mtu")
+
+// Plugin wraps the DNS plugin information.
+var Plugin = plugins.Plugin{
+	Name:   "mtu",
+	Setup4: setup4,
+}
```

Comment:
a small copy-paste mistake. I'd also like to add a comment mentioning that this is not relevant for dhcpv6 (to distinguish from plugins where we just haven't implemented it yet)
```suggestion
// Plugin wraps the MTU plugin information.
var Plugin = plugins.Plugin{
	Name:   "mtu",
	Setup4: setup4,
	// No Setup6 since DHCPv6 does not have MTU-related options
}
```

Reply:
it's done.